collector/etherscan.go

```diff
@@ -133,26 +161,34 @@ func (c EtherscanClient) getPagedTransactions(address Address, page, offset int)
 		return
 	}
 
+	latestBlock := c.getLatestBlock()
+
 	q := req.URL.Query()
 	q.Add("module", "account")
 	q.Add("action", "txlist")
 	q.Add("address", string(address))
+	q.Add("startblock", fmt.Sprint(latestBlock - 25))
```

Comment:
Can we make this an ENV parameter, so we could try to increase it without de-deploying?